src/sections/trade/TradePage.tsx

```diff
@@ -1,123 +1,32 @@
-
-  const handleSubmit = async (e: CustomEvent<TxInfo>) => {
-    const { transaction, notification } = e.detail
-    await createTransaction(
-      {
-        tx: api.tx(transaction.hex),
-      },
-      {
-        onSuccess: () => {},
-        onSubmitted: () => {},
-        toast: {
-          onLoading: (
-            <span
-              dangerouslySetInnerHTML={{
-                __html: notification.processing.rawHtml,
-              }}
-            />
-          ),
-          onSuccess: (
-            <span
-              dangerouslySetInnerHTML={{
-                __html: notification.success.rawHtml,
-              }}
-            />
-          ),
-          onError: (
-            <span
-              dangerouslySetInnerHTML={{
-                __html: notification.failure.rawHtml,
-              }}
-            />
-          ),
-        },
-      },
-    )
-  }
-
+import { LINKS } from "utils/navigation"
+import { SwapPage } from "./sections/swap/SwapPage"
+import { useMatchRoute } from "@tanstack/react-location"
+import { BondsPageWrapper } from "./sections/bonds/BondsPageWrapper"
+import { OtcPageWrapper } from "./sections/otc/OtcPageWrappet"
+import { DcaApp } from "./sections/dca/DcaPage"
+import { SubNavigation } from "./SubNavigation"
+
+const isOtcPageEnabled = import.meta.env.VITE_FF_OTC_ENABLED === "true"
+const isDcaPageEnabled = import.meta.env.VITE_FF_DCA_ENABLED === "true"
+const isBondsPageEnabled = import.meta.env.VITE_FF_BONDS_ENABLED === "true"
+
+export const TradePage = () => {
+  const matchRoute = useMatchRoute()
   return (
-    <Page>
-      <SContainer>
-        <TradeApp
-          ref={(r) => {
-            if (r) {
-              r.setAttribute("chart", "")
-              isTwapEnabled && r.setAttribute("twap", "")
-            }
-          }}
-          assetIn={search.success ? search.data.assetIn : undefined}
-          assetOut={search.success ? search.data.assetOut : undefined}
-          apiAddress={rpcUrl}
-          pools={PoolType.Omni}
-          stableCoinAssetId={stableCoinAssetId}
-          accountName={account?.name}
-          accountProvider={account?.provider}
-          accountAddress={account?.address}
-          indexerUrl={indexerUrl}
-          grafanaUrl={grafanaUrl}
-          grafanaDsn={grafanaDsn}
-          onTxNew={(e) => handleSubmit(e)}
-          onDcaSchedule={(e) => handleSubmit(e)}
-          onDcaTerminate={(e) => handleSubmit(e)}
-        />
-      </SContainer>
+    <Page
+      subHeader={<SubNavigation />}
+      subHeaderStyle={{
+        background: "rgba(9, 9, 9, 0.09)",
+        borderTop: "1px solid rgba(114, 131, 165, 0.6)",
+      }}
+    >
+      {matchRoute({ to: LINKS.swap }) && <SwapPage />}
+      {isBondsPageEnabled && matchRoute({ to: LINKS.bonds }) && (
+        <BondsPageWrapper />
+      )}
+      {isOtcPageEnabled && matchRoute({ to: LINKS.otc }) && <OtcPageWrapper />}
+      {isDcaPageEnabled && matchRoute({ to: LINKS.dca }) && <DcaApp />}
```

Comment:
I think we should avoid this manual routing. If you use `<Outlet />` component in here and then define routes as follows: 

```
{
    path: "trade",
    element: <TradePage />,
    children: [
      {
        path: "/",
        element: <Navigate to="swap" />,
      },
      {
        path: "swap",
        element: <SwapPage />,
      },
      {
        ...(isOtcPageEnabled && {
          path: "otc",
          element: <OtcPageWrapper />,
        }),
      },
      ...
    ],
  },

```

you should get a cleaner version without the need to route manually